.jenkins/jenkins_buildbot_python2_debug.sh

```diff
@@ -10,7 +10,7 @@ export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
 export LIBRARY_PATH=/usr/local/cuda/lib64:$LIBRARY_PATH
 
 GPUARRAY_CONFIG="Release"
-DEVICE=cuda0
```

Comment:
It was used by the libgpuarray buildbot to do the cmake libgpurray and pygpu tests, which we don't do here. We can remove the line in another PR.